pkgs/servers/uxplay/default.nix

```diff
@@ -14,23 +14,15 @@
 
 stdenv.mkDerivation rec {
   pname = "uxplay";
-  version = "1.50";
+  version = "1.52";
 
   src = fetchFromGitHub {
     owner = "FDH2";
     repo = "UxPlay";
     rev = "v${version}";
-    sha256 = "sha256-43BCpYh0XtsnI064/ddcz2/Imj399g+bxLlT0BpqLMI=";
+    sha256 = "sha256-2wPUG50fbXLg6w2Rni3NyeiCyUNPcOvxvqopD9QZJaQ=";
   };
 
-  patches = [
-    # https://github.com/FDH2/UxPlay/issues/91
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done